src/class-restapi.php

```diff
@@ -168,6 +208,78 @@ public static function add_results_callback( $data ) {
 		return $response;
 	}
 
+	private static function get_new_failures( $post_id ) {
+		$p         = get_post( $post_id );
```

Comment:
This is a really neat idea!

I think there are some potential issues here though. I don't think we can query for the intended post with the details here. If an old report is visited in the future, I believe that this would compare that one to the most recent report.

I think a better way to ensure this works correctly is to look up the previous commit's parent using a `get_post_by( 'slug', 'r' . $commit_number - 1 );` call. Then we can specify `parent_in` instead of `parent_not_in` with the other differentiating meta to ensure the right results are compared.

Reply:
> If an old report is visited in the future, I believe that this would compare that one to the most recent report.

You mean when sending results for an older commit, while a new one already exists? Mhm yeah might need some testing here.

> think a better way to ensure this works correctly is to look up the previous commit's parent using a `get_post_by( 'slug', 'r' . $commit_number - 1 );` call

It's not guaranteed that an entry exists for the previous commit.